packages/transformers/js/src/JSTransformer.js

```diff
@@ -442,6 +442,7 @@ export default (new Transformer({
       is_esm_output: asset.env.outputFormat === 'esmodule',
       trace_bailouts: options.logLevel === 'verbose',
       is_swc_helpers: /@swc[/\\]helpers/.test(asset.filePath),
+      standalone: asset.query.has('standalone'),
```

Comment:
Not sure about using a query param to drive this behaviour. Maybe this should be part of the environment?

Reply:
I did think about it, for now using query provides some flexibility & less moving part to change the environment. Kinda like a feature flag for `jsTransform` for now.